src/test/regress/sql/citus_stat_tenants.sql

```diff
@@ -311,5 +311,56 @@ SELECT count(*)>=0 FROM select_from_dist_tbl_text_view WHERE a = U&'\0061\0308bc
 
 SELECT tenant_attribute, query_count_in_this_period FROM citus_stat_tenants;
 
+-- schema based tenants
```

Comment:
We should add a simple test to see that queries are not tracked when the query is on a single shard distributed table, which is not part of a tenant schema `create_distributed_table('t1', NULL)`. A very simple test is fine, just like the one we have for reference tables now.